R/readTechCosts.R

```diff
@@ -809,9 +811,9 @@ readTechCosts <- function(subtype = "PowerAndHeat") { # nolint
     x <- matrix(NA, 64, 5)
     x <- as.data.frame(x)
     x[seq(from=1, to=64, by=2) , 4] <- df[,4]
-    x[seq(from=1, to=64, by=2) , 5] <- ("(Euro/Household)")
+    x[seq(from=1, to=64, by=2) , 5] <- word(names(df)[4], 3, 3)

```

Comment:
why the use of `word` here? importing too many packages is not good practice. is there no alternative?

Reply:
use of gsub instead of word